obal/data/modules/check_copr_build.py

```diff
@@ -0,0 +1,54 @@
+"""
+Check if a build already exists in Copr
+"""
+
+import json
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.copr import copr_cli, CoprCliCommandError, full_name # pylint:disable=import-error,no-name-in-module
+
+
+def main():
+    """
+    Check if a build already exists in Copr
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            user=dict(type='str', required=True),
+            project=dict(type='str', required=True),
+            nvr=dict(type='str', required=True),
+            package=dict(type='str', required=True),
+            config_file=dict(type='str', required=False),
+        )
+    )
+
+    user = module.params['user']
+    project = module.params['project']
+    nvr = module.params['nvr']
+    package = module.params['package']
+    config_file = module.params['config_file']
+
+    command = [
+        'get-package',
+        full_name(user, project),
+        '--name',
+        package,
+        '--with-all-builds'
+    ]
+
+    try:
+        package_info = json.loads(copr_cli(command, config_file=config_file))
+    except CoprCliCommandError as error:
+        if "Error: No package with name {} in copr {}".format(package, project):
+            module.exit_json(changed=True, exists=False)
+        else:
+            module.fail_json(msg='Retrieval of package from Copr failed', command=command, output=error.message,
+                             repo_name=full_name(user, project), package=package)
+
+    builds = (build for build in package_info['builds'] if build['state'] == 'succeeded')
+    exists = any("{}-{}".format(package, build['source_package']['version']) == nvr for build in builds)
+
+    module.exit_json(changed=not exists, exists=exists)
```

Comment:
I find it weird that a `check_*` module has a `changed` flag -- it did not change anything.
And you don't use that in the playbooks later either, just `exists` -- so I'd just drop it here (and in the other `exit` calls)